app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/data/base/JsonEntity.java

```diff
@@ -0,0 +1,17 @@
+package com.duckduckgo.mobile.android.duckduckgo.data.base;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public abstract class JsonEntity {
```

Comment:
I prefer the interface too, but we need the empty constructor to ensure SharedPreferencesDataStore can instantiate the JsonEntity. With just an interface can we be sure each entity will have a empty constructor? (I'm asking because I was stack a few minutes with the store not working correctly)